modules/dds/register_types.cpp

```diff
@@ -33,10 +33,12 @@
 #include "image_saver_dds.h"
 #include "texture_loader_dds.h"
 
+#include "scene/resources/texture.h"
+
 static Ref<ResourceFormatDDS> resource_loader_dds;
 
 void initialize_dds_module(ModuleInitializationLevel p_level) {
-	if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE) {
+	if (p_level != MODULE_INITIALIZATION_LEVEL_SCENE || !GD_IS_CLASS_ENABLED(Texture)) {
```

Comment:
In other modules you used a different pattern with `if (GD_IS_CLASS_ENABLED(Texture)) {}` enclosing the instantiation of the ResourceLoader. I think it's best to keep it consistent, so I'd standardize like that.

And actually, that would fix a bug here, as the `Image` function pointers should still be kept, they don't depend on `Texture`.

Reply:
Done.